src/content/app/new-species-selector/components/selectable-genomes-table/useOrderedGenomes.ts

```diff
@@ -0,0 +1,198 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useState, useCallback } from 'react';
+import type { SelectableGenome } from './useSelectableGenomesTable';
+
+type SortableColumn =
+  | 'common_name'
+  | 'scientific_name'
+  | 'type'
+  | 'assembly_name'
+  | 'assembly_accession_id'
+  | 'coding_genes_count'
+  | 'contig_n50';
+
+type SortOrder = 'asc' | 'desc';
```

Comment:
Shouldn't `SortOrder` also have `none` as a value?

Reply:
No.

I've been struggling with the way how to represent "no sort order". For the ColumnHeader component, I was debating between `'asc' | 'desc' | 'none'` and `'asc' | 'desc' | null`; and managed to convince myself that if `ColumnHeader` were an html element, then the string 'none', which can be passed as an attribute, would be preferable to `null`, which only has to be passed as a property. I am still not sure whether the interface I chose for `ColumnHeader` is correct.

But for the hook, there is no need for the `none` string — it is responsible for determining the sort order by listening for signals from all columns; and there may only be either a sorting rule for one of the columns (ascending or descending), or no sorting rule at all (i.e. null).

Reply:
On the other hand, `SortOrder` type is defined in quite a few places and seems to be almost of the same type (except for `none` not used in a couple of places). Can it be reused rather than being defined everywhere?

Reply:
Ok, I created two types, `SortOrder` and `SortOrderWithNone`, and moved them into a dedicated file.